### ISSUE TITLE
sv can not open files with ':' in filename 

### ISSUE THREAD

**Issue by kermitfrog** [roboception/cvkit#15]
After hours of trying to find out what is wrong with my generated pfm files, and getting the following error message:

```
X Error of failed request:  BadValue (integer parameter out of range for operation)
  Major opcode of failed request:  12 (X_ConfigureWindow)
  Value in failed request:  0x0
  Serial number of failed request:  26
  Current serial number in output stream:  30
```

I found the answer: absolutely nothing!

\*sigh\*

sv just couldn't open it, because the filename contained a ':' character 

this is on linux btw.

**Comment by heikohimu** [comment]
I tracked the source of the problem. In cvkit, I implemented working with arbitrarily large images that are stored in tiles. I needed that for working with ortho images that were assembled from thousands of aerial images in my former job at the German Aerospace Center. For (my) convenience, I implemented specifying tiled images as <prefix>:<suffix> (see USAGE.md). This means the ':' character is tells sv that this is a file name of a tiled image. It never came to my mind that this character would be used in a 'normal' file name, especially since under Windows it also has a special meaning. The strange error message actually appears, because sv in the end cannot find the tiled image and allocates an image of size 0x0, which cannot be displayed.

Sorry for the inconvenience. I will at least improve the error message!
